schemas/com.iab.snowplow/iab_spiders_and_robots/jsonschema/1-0-0

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for IAB Robots&Spiders POJO",
+    "self": {
+        "vendor": "com.iab.snowplow",
+        "name": "iab_spiders_and_robots",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+
+    "type": "object",
+    "properties": {
+        "spiderOrRobot": {
```

Comment:
It is not that clear who (`spider` or `robot`) should be `true` and who should be `false`. Adding at least description would be useful.

Reply:
It's actually `true` if the IP address/user agent is detected as a "spider" _or_ "robot" (which aren't formally separated either-or categories, the list just contains "spiders and robots") and `false` otherwise - I can see how the field name could be a bit confusing. I've added some descriptions for this and the enrichment schema.

Reply:
Yep, totally makes sense now. Thanks.